.github/workflows/test-unit.yml

```diff
@@ -0,0 +1,62 @@
+---
+name: Test unit
+
+on:
+  push:
+    branches:
+      - main
+      - master
+      - develop
+      - update-external-dependencies
```

Comment:
Remove the branches that do not exist. Same in the `lint.yml` file. WDYT?
```suggestion
```

Reply:
Fix: https://github.com/0xPolygon/zkevm-ethtx-manager/pull/61/commits/d122c99a1a814a992571982b10bcfc0941d954a2